v6/go.mod

```diff
@@ -3,17 +3,123 @@ module github.com/GoogleCloudPlatform/terraform-google-conversion/v6
 go 1.23
 
 require (
+	github.com/hashicorp/terraform-provider-google-beta v1.20.1-0.20241219201437-6fd519c88f0b
```

Comment:
The TPGB version is hardcoded for now. Will move it later after the schemas are generated to TGC.